src/main/java/org/mvplugins/multiverse/core/api/BlockSafety.java

```diff
@@ -37,43 +38,74 @@ public interface BlockSafety {
      */
     boolean playerCanSpawnHereSafely(Location l);
 
+    /**
+     * Finds a portal-block next to the specified {@link Location}.
+     * @param location  The {@link Location}
+     * @return The next portal-block's {@link Location}.
+     */
+    Location findPortalBlockNextTo(Location location);
+
+    /**
+     * Gets the next safe location around the given location.
+     *
+     * @param location  A {@link Location}.
+     * @return A safe {@link Location}.
+     */
+    @Nullable Location getSafeLocation(Location location);
+
+    /**
+     * Gets the next safe location around the given location.
+     *
+     * @param location  A {@link Location}.
+     * @param tolerance The tolerance.
+     * @param radius    The radius.
+     * @return A safe {@link Location}.
+     */
+    @Nullable Location getSafeLocation(Location location, int tolerance, int radius);
```

Comment:
Could you add some info about what tolerance is in these docs?

Reply:
yea sure... this method is just moved, so didnt really check it